packages/accordion/src/hooks/useAccordionAnimation.ts

```diff
@@ -0,0 +1,37 @@
+import { useLayoutEffect, useRef, useState } from 'react';
+
+export const useAccordionAnimation = (isOpen: boolean) => {
+  const ref = useRef<HTMLDivElement>(null);
+  const [height, setHeight] = useState<string>('0px');
```

Comment:
```suggestion
  const [height, setHeight] = useState<CSSProperties['height']>('0px');
```

타입안정성을 위해서 CSSProperties가져오는건 어떨까요 ?

Reply:
@froggy1014 해당 부분은 리팩토링할때 반영된 코드를 통해 개선할게! state를 사용하는 대신 관심사 분리를 통해 CSS 변수로 떼어내서 해당 커스텀 훅에서는 높이를 측정하는 역할만 하려고해.